<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>azienda</class>
 <widget class="QDialog" name="azienda">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>397</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Azienda</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../biblio.qrc">
    <normaloff>:/images/logo_p32.png</normaloff>:/images/logo_p32.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Anagrafica azienda</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="6">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="4" colspan="2">
       <widget class="QLineEdit" name="prov"/>
      </item>
      <item row="3" column="2" colspan="4">
       <widget class="QLineEdit" name="p_iva"/>
      </item>
      <item row="11" column="4" colspan="2">
       <widget class="QLineEdit" name="fax"/>
      </item>
      <item row="5" column="2" colspan="4">
       <widget class="QLineEdit" name="cod_f"/>
      </item>
      <item row="13" column="4" colspan="2">
       <widget class="QLineEdit" name="email"/>
      </item>
      <item row="11" column="1" colspan="2">
       <widget class="QLineEdit" name="telef"/>
      </item>
      <item row="11" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Fax</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="6">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Telefono</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Partita IVA</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>PROV</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Indirizzo</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item row="14" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="localita"/>
      </item>
      <item row="13" column="1" colspan="2">
       <widget class="QLineEdit" name="web"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Codice fiscale</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="6">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLineEdit" name="indirizzo"/>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QLineEdit" name="rag_soc"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Sito web</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4" colspan="2">
       <widget class="QLineEdit" name="cap"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ragione sociale</string>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizeIncrement">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Elimina azienda</string>
        </property>
        <property name="text">
         <string>Elimina</string>
        </property>
        <property name="icon">
         <iconset resource="../biblio.qrc">
          <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CAP</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Localit√†</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="6">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="14" column="5">
       <widget class="QPushButton" name="pushButton">
        <property name="toolTip">
         <string>Esci da anagrafica 
azienda</string>
        </property>
        <property name="text">
         <string>Chiudi</string>
        </property>
        <property name="icon">
         <iconset resource="../biblio.qrc">
          <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QPushButton" name="licenza">
        <property name="toolTip">
         <string>Salva azienda</string>
        </property>
        <property name="text">
         <string>Salva</string>
        </property>
        <property name="icon">
         <iconset resource="../biblio.qrc">
          <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rag_soc</tabstop>
  <tabstop>p_iva</tabstop>
  <tabstop>cod_f</tabstop>
  <tabstop>indirizzo</tabstop>
  <tabstop>cap</tabstop>
  <tabstop>localita</tabstop>
  <tabstop>prov</tabstop>
  <tabstop>telef</tabstop>
  <tabstop>fax</tabstop>
  <tabstop>web</tabstop>
  <tabstop>email</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../biblio.qrc"/>
 </resources>
 <connections/>
</ui>
